config/default.yml

```
@@ -4,6 +4,11 @@ Minitest:
     - '**/test/**/*'
     - '**/*_test.rb'
 
+Minitest/AnyAssertions:
+  Description: 'This cop checks for at least one assertion (or flunk) in tests.'
+  Enabled: 'pending'
+  VersionAdded: '0.11'
```

My review was delayed and RuboCop Minitest 0.11 has been released. Can you update to `0.12`?

```suggestion
  VersionAdded: '0.12'
```